test/quantization/test_quantize.py

```diff
@@ -447,6 +447,49 @@ def checkQuantized(model):
                                  self.calib_data)
         checkQuantized(model_oneline)
 
+    def test_forward_hooks_preserved(self):
+        r"""Test post-training static quantization on preserving 
+        pre forward and post forward hooks of original model
+        """
+        for qengine in supported_qengines:
+            with override_quantized_engine(qengine):
```

Comment:
you can use this decorator: https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_quantized.py#L119